<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QDialog" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>238</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>姓名</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="accountLabel">
         <property name="text">
          <string>注册账号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="passwordLabel">
         <property name="text">
          <string>输入密码</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="passwordLabel1">
         <property name="text">
          <string>再次输入</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="secretKeyLabel">
         <property name="text">
          <string>密钥</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="name"/>
       </item>
       <item>
        <widget class="QLineEdit" name="account"/>
       </item>
       <item>
        <widget class="QLineEdit" name="password"/>
       </item>
       <item>
        <widget class="QLineEdit" name="password1"/>
       </item>
       <item>
        <widget class="QLineEdit" name="secretKey"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="staffBtn">
       <property name="text">
        <string>员工</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="manageBtn">
       <property name="text">
        <string>管理员</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
